risc0/zkvm/examples/loop.rs

```diff
@@ -26,6 +26,15 @@ use risc0_zkvm_methods::{
 };
 use tracing_subscriber::{prelude::*, EnvFilter};
 
+#[derive(serde::Serialize, Debug)]
+struct PerformanceData {
+    cycles: String,
+    duration: String,
+    ram: String,
+    seal: String,
+    speed: String,
+}
```

Comment:
Ideally the json data would use numbers rather than strings so that we can use this data in other ways in the future. We could make all the human readable stuff happen at a later phase perhaps in client-side javascript.